packages/manager/apps/web/client/app/hosting/cdn/order/hosting-cdn-order.controller.js

```diff
@@ -1,56 +1,109 @@
 import filter from 'lodash/filter';
 import includes from 'lodash/includes';
+import get from "lodash/get";
 
 export default class {
   /* @ngInject */
-  constructor($filter, $timeout) {
+  constructor($scope, $filter, $timeout, $translate, HostingCdnOrderService) {
+    this.$scope = $scope;
     this.$filter = $filter;
     this.$timeout = $timeout;
+    this.$translate = $translate;
+    this.HostingCdnOrderService = HostingCdnOrderService;
+    this.cdnCase = null;
+    this.cdn1Price = null;
   }
 
   $onInit() {
     // Auto-select duration
+    console.log('ZM:: serviceInfo', this.serviceInfo);
+    console.log('ZM:: catalogAddon', this.catalogAddon);
+    console.log('ZM:: catalogAddons', this.catalogAddons);
```

Comment:
If still in progress maybe it should be a draft

Reply:
yes i let a comment to ignore comments and logs